### PR TITLE
feat(Core/Config): Arena points - Games required

### PR DESCRIPTION
**Pull request by aradep**
## CHANGES PROPOSED:
Adds option to worldserver.conf for setting the number of arena matches teams must participate in to be eligible for arena point distribution.

## HOW TO TEST THE CHANGES:
Change ```Arena.GamesRequired``` in ```worldserver.conf``` to 1
Play 1 arena game
.flusharenapoints

## Target branch(es):
- [x] Master

## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Review by FrancescoBorzi — APPROVED** [review]
very nice idea, good job

File: src/server/worldserver/worldserver.conf.dist [hunk 1]
```diff
@@ -2936,6 +2936,13 @@ Arena.AutoDistributePoints = 0
 
 Arena.AutoDistributeInterval = 7
 
+#
+#    Arena.GamesRequired
+#        Description: Number of arena matches team must participate in to be eligible for arena point distribution.
```
Comment (BarbzYHOOL) [inline src/server/worldserver/worldserver.conf.dist:2941]:
```suggestion
#        Description: Number of arena matches teams must participate in to be eligible for arena point distribution.
```
Number or amount? i don't remember

Comment (aradep) [reply src/server/worldserver/worldserver.conf.dist:2941]: both wordings are used about equally in the .conf

**Comment by nicoaravena** [comment]
Hey there!
Just tested this PR.
Arenas works fine, without errors.
Configuration was setted as recomended for testing `Arena.GamesRequired = 1`
Score before battle
imagen

Score after battle:
imagen

Arena points after `.flusharenapoints`
imagen

Also change the config to `Arena.GamesRequired = 2`, play one match and then to `.flusharenapoints` and no points added as spected.

**Comment by aradep** [comment]
@nicoaravena Great job, thanks for the test!

**Comment by FrancescoBorzi** [comment]
good job @aradep and @nicoaravena !

**Review by Si1ker — APPROVED** [review]

**Pull request merged**